### PR TITLE
BaSP-MR-28: tests admin methods get and update

### PR DESCRIPTION
**Pull request by Eugeniagassman**
Unit test with jest to validate the functionality of the get, getById and update methods.
I also created a seed database to be able to test without modifying or damaging the mongoo database.
 In the test of the get method includes seeing the code states and validating certain data in the database created in seeds. It also looks for a data by id, it tests the method when passing an invalid id, or an id that does not exist.
In the test of the put method it is validated that a data has been updated correctly using an id, in addition it is tested when passing an invalid id or an id that does not exist in our database in our folder.

### CODE REVIEW

**Review by GaloDurante — APPROVED** [review]
Looks good to me!

**Review by NicolasKler — APPROVED** [review]
Seems good to me

File: src/tests/admins.test.js [hunk 1]
```diff
@@ -1,3 +1,65 @@
...
+  test('should throw error for a wrong path and status 404', async () => {
+    const res = await request(app).get('/api/admin').send();
+    expect(res.status).toBe(404);
+    expect(res.error).toBeTruthy();
+  });
+  test('should get one admin by id', async () => {
+    const res = await request(app).get('/api/admins/64654f260425c5c72ff190fe').send();
+    // eslint-disable-next-line no-underscore-dangle
+    expect(res.body.data._id).toEqual('64654f260425c5c72ff190fe');
+    expect(res.body.data.firstName).toEqual('Euge');
```
Comment (FontanR) [inline src/tests/admins.test.js:25]: maybe you could validate email or dni instead of name

File: src/tests/admins.test.js [hunk 2]
```diff
@@ -1,3 +1,65 @@
...
+describe('PUT/api/admins/:id', () => {
+  test('you must update an admin', async () => {
+    const updAdmin = { firstName: 'Juan', lastName: 'Admin', dni: '123456789' };
+    const response = await request(app).put('/api/admins/646550c10425c5c72ff19102').send(updAdmin);
+    expect(response.status).toBe(200);
+    expect(response.body.error).toBeFalsy();
+    const res = await request(app).get('/api/admins/646550c10425c5c72ff19102').send();
+    // eslint-disable-next-line no-underscore-dangle
+    expect(res.body.data._id).toEqual('646550c10425c5c72ff19102');
+    expect(res.body.data.firstName).toEqual('Juan');
```
Comment (FontanR) [inline src/tests/admins.test.js:50]: same here

**Review by fernandofernandezuy — APPROVED** [review]
Looks good to me!

**Pull request merged**